@xen-orchestra/web-core/lib/components/fullscreenable/FullscreenableHeader.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <header class="header">
```

Comment:
According to the design system, the content of the header should have the `p2-medium` font-size / font-weight.
Also, by convention the root element's class name should be `fullscreenable-header`:
```suggestion
  <header class="fullscreenable-header typo p2-medium">
```